BlogTemplate/Pages/Edit.cshtml.cs

```diff
@@ -58,13 +58,31 @@ public IActionResult OnPostSaveDraft()
         public void UpdatePost(Post newPost, string slug)
         {
             oldPost = _dataStore.GetPost(slug);
+            DateTime OrigPubDate = oldPost.PubDate;
+            // Body and excerpt are not updated either. They are whatever they were inputted into the form as
+            newPost.PubDate = OrigPubDate;
```

Comment:
OrigPubDate seems unnecessary, you can just new `newPost.PubDate = oldPost.PubDate`